changelog.md

```diff
@@ -5,7 +5,41 @@ All notable end-user facing changes are documented in this file.
 *Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
 The idea is to track end-user facing changes as they occur.*
 
-### [v1.4.2]
+### [1.5.0-rc]  (changes since 1.5.0-beta1)
+- Breaking changes
+  - Bug fix: Azure storage providers now throw `InconsistenStateException` instead of `StorageException` when eTags do not match #2971
+  - Renamed the `Catalog.Activation.DuplicateActivations` counter to `Catalog.Activation.ConcurrentRegistrationAttempts` to more accurately reflect what it tracks and its benign nature #3130
+  - Upgraded `WindowsAzure.ServiceBus` package dependency to 4.1.0 #3127
+  - Replaced `CacheSizeInMb` setting with `DataMaxAgeInCache` and `DataMinTimeInCache` in stream providers #3126
+- Non-breaking improvements
+  - Enable runtime policy change for Silo versioning #3055
+  - Add support for hash-based grain placement #2944
+  - Allow complex streaming filters in `ImplicitStreamSubscriptionAttribute` #2988
+  - Support fire and forget one-way grain calls using `[OneWay]` method attribute #2993
+  - Support exceptions with reference cycles in ILBasedExceptionSerializer #2999
+  - Add extensibility point to replace the grain activator and finalizer #3002
+  - Allow `IGrainWithGuidCompoundKey` as implicit subscription grain #3011
```

Comment:
I just tagged streaming PRs with breaking change label. btw. But I think it maybe a good process to apply to PRs in other areas. since people has poor habit to remember updating the change log.